src/main/resources/data/content/paymentTemplate.hbs

```diff
@@ -0,0 +1,229 @@
+<html lang="en">
+
+<head>
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1"/>
+  <title>Proof of Payment</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com"/>
+  <link rel="preconnect" href="https://fonts.gstatic.com"/>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700" rel="stylesheet"/>
```

Comment:
Since the html is being converted to pdf on the server, will these links be resolved on the server?
Typically services should not be able to access external content. (see recent liquibase xsd issue)